test/libsolidity/semanticTests/multiSource/_external/import.sol

```diff
@@ -0,0 +1,2 @@
+import "external.sol";
+import "otherExternal.sol";
```

Comment:
I think it is ok to support only the import of subsequent external files like this. In such file we just list all the external dependencies that need to be imported. I could also try to resolve all imports recursively, but I think that this is not really needed. What do you think?

Reply:
I think it is fine like this for now. If needed, we can resolve imports recursively in another PR.